src/maptm/src/maptm.h

```diff
@@ -38,7 +38,10 @@
 #include "osp.h"
 #include "osp_ps.h"
 
-#define MAPTM_MODULE_NAME           "MAPTM"
+#define MAPT_MODULE_NAME           "MAPT"
+
+#define MAPT_SUPPORT_STORE "mapt"
+#define MAPT_SUPPORT_VALUE "elected"
```

Comment:
```suggestion
#define MAPT_SUPPORT_STORE  MAPT_MODULE_NAME  /**< store name == module name */
#define MAPT_SUPPORT_VALUE  "enabled"
```

Previous suggestion wasn't well thought through, we should keep names in Node_Config and persistent store in sync as much as possible.